pages/convince-your-boss.tsx

```diff
@@ -0,0 +1,170 @@
+import Link from "next/link";
+import CenteredButton from "../ui/centered-button";
+import Divider from "../ui/divider";
+import ColoredSection from "../ui/colored-section";
+import SocialShare from "../ui/social-share";
+
+function CallForPapers() {
+  return (
+    <>
+      <SocialShare
+        title="Convince your boss!"
+        description=""
+        socialShareImage="social-share/convince-your-boss.png"
```

Comment:
"Did you mean to add an attachment?" - displays in gmail popup :)

Reply:
fixed!